src/client/components/App.tsx

```diff
@@ -70,6 +75,11 @@ const App: FunctionComponent = (): ReactElement => {
    */
   const [isDataFetching, setDataFetching] = useState(true);
 
+  /**
+   * Tracks the current version of the application to be displayed in the footer
+   */
+  const [appVersion, setAppVersion] = useState('');
```

Comment:
Since the APP_VERSION variable is never _really_ going to change, I don't think we actually need `useState` here. I think it might make more sense to just declare it outside the scope of the Component (i.e. above line 42) as

```ts
const APP_VERSION = process.env.APP_VERSION;
```
(Note that we can't destructure the value from `process.env`, because we're relying on Webpack to do a simple string replacement; in the compiled code this would be replaced with `const APP_VERSION = "v0.14.1";`)


It's not necessarily a bad thing to make it a state value, but I think doing so implies that there are circumstances where it might change dynamically, when there really aren't.

Reply:
Thanks Jonathan! That's a good point about the version not changing dynamically. I updated the code to declare the application version above the `App` component. I ended up sticking with the variable name `appVersion` so that I did not have to disable the "prefers destructuring" linting rule. (commit [1026ee5](https://github.com/seas-computing/course-planner/pull/631/commits/1026ee5091e293ec9cd8da5d327cc99641a6de5e))